services/schedule_create.proto

```diff
@@ -90,4 +100,33 @@ message ScheduleCreateTransactionBody {
    * given
    */
   AccountID payerAccountID = 4;
+
+  /**
+   * An optional timestamp for specifying when the transaction should be evaluated for execution and then expire.
+   * Defaults to 30 minutes after the transaction's consensus timestamp.
+   *
+   * Note: This field is unused and forced to be unset until Long Term Scheduled Transactions are enabled - Transactions will always
+   *       expire in 30 minutes if Long Term Scheduled Transactions are not enabled.
+   */
+  Timestamp expirationTime = 5;
+
+  /**
+   * When set to true, the transaction will be evaluated for execution at expirationTime instead
+   * of when all required signatures are received.
+   * When set to false, the transaction will execute immediately after sufficient signatures are received
+   * to sign the contained transaction. During the initial ScheduleCreate transaction or via ScheduleSign transactions.
+   *
+   * Defaults to false.
+   *
+   * Setting this to false does not necessarily mean that the transaction will never execute at expirationTime.
+   *  For Example - If the signature requirements for a Scheduled Transaction change via external means (e.g. CryptoUpdate)
+   *  such that the Scheduled Transaction would be allowed to execute, it will do so autonomously at expirationTime, unless a
+   *  ScheduleSign comes in to “poke” it and force it to go through immediately.
+   *
+   * Note: This field is unused and forced to be unset until Long Term Scheduled Transactions are enabled. Before Long Term
```

Comment:
I have a concern about adding fields to the protobuf if we don't intend to use them soon. We've done this a lot in the past and the API is littered with unused fields. Even worse is if we add them then later find out we need something different like `proxyAccountId` for staking. If this isn't going in 0.26 then we should consider not putting these in the protobuf until later.

Reply:
I'm very sure this is what we need. If we want to wait for 0.27 I guess we can. @SimiHunjan ?

Reply:
Can we make the changes in the feature branch and merge to develop/main when ready for the 0.27 release cycle?

Reply:
Had call with @steven-sheehy and others, we decided that since this is going in 0.26 it can be merged now.